File: Lua/primes.lua
start = os.clock()

primes = {}

for i = 0, 15000 do
	factors = {}

	for j = 1, i do
		if i % j == 0 then
			table.insert(factors, j)
		end
	end

	if #factors == 2 then
		table.insert(primes, i)
	end
end

print("["..table.concat(primes, ", ").."]")

print(string.format("Time taken: %.2fs\n", os.clock() - start))

File: Php/primes.php
<?php

$time1 = time();
$primes = [];

for ($i = 0; $i < 15000; $i++) {
    $factors = [];

    for ($j = 1; $j <= $i; $j++) {
        if ($i % $j == 0) {
            array_push($factors, $j);
        }
    }

    if (count($factors) == 2) {
        array_push($primes, $i);
    }
}

var_dump($primes);
$time2 = time();
$performance = $time2 - $time1;
echo "Time taken: $performance s";
